dissect/target/plugins/apps/browser/iexplore.py

```diff
@@ -192,12 +193,20 @@ def downloads(self) -> Iterator[BrowserDownloadRecord]:
         """
         for user, cache_file, cache in self._iter_cache():
             for container_record in cache.downloads():
-                response_headers = container_record.ResponseHeaders.decode("utf-16-le", errors="ignore")
-                ref_url, mime_type, temp_download_path, down_url, down_path = response_headers.split("\x00")[-6:-1]
+                down_path = None
+                down_url = None
+                ts_end = wintimestamp(container_record.AccessedTime) if container_record.AccessedTime else None
+
+                try:
+                    response_headers = container_record.ResponseHeaders.decode("utf-16-le", errors="ignore")
+                    ref_url, mime_type, temp_download_path, down_url, down_path = response_headers.split("\x00")[-6:-1]
```

Comment:
To prevent future bugs, could you either initialize `ref_url`, `mime_type` and `temp_download_path` to `None`? Or replace those with `_` and leave a comment explaining what those fields would be.

Reply:
Good one, changed it! Complemented the existing test as well, to prevent regression with the path/url value